LanguageFeatures/Augmentation-libraries/augmenting_types_A01_t03.dart

```diff
@@ -26,12 +25,8 @@ typedef CAlias = C;
 typedef MAlias = M;
 typedef EAlias = E;
```

Comment:
Is it true that these three declarations could just be removed? I don't see them in use anywhere.

Reply:
Right. Thank you! Removed